DemoApp/StreamChat/Components/DemoChatChannelListRouter.swift

```diff
@@ -447,6 +447,22 @@ final class DemoChatChannelListRouter: ChatChannelListRouter {
                     )
                 ), animated: true)
             }),
+            .init(title: "Show Blocked Users", handler: { [unowned self] _ in
+                guard let cid = channelController.channel?.cid else { return }
+                let client = channelController.client
+                client.currentUserController().getBlockedUsers { result in
+                    if let blockedUsers = try? result.get() {
+                        self.rootViewController.present(MembersViewController(
+                            membersController: client.memberListController(
+                                query: .init(
+                                    cid: cid,
+                                    filter: .in(.id, values: blockedUsers.map({ $0.userId }))
+                                )
+                            )
+                        ), animated: true)
+                    }
```

Comment:
Nit: a little be easier to read if we use guard here. Otherwise, it is too much nested blocks
```suggestion
                    guard let blockedUsers = try? result.get() else { return }
                    self.rootViewController.present(MembersViewController(
                        membersController: client.memberListController(
                            query: .init(
                                cid: cid,
                                filter: .in(.id, values: blockedUsers.map({ $0.userId }))
                            )
                        )
                    ), animated: true)
```